assets/scss/main.scss

```diff
@@ -36,6 +38,11 @@
   }
 }
 
+.square {
+  width: 100px;
+  height: 100px;
+}
```

Comment:
we can accomplish this using a grid layout. and have it be flexible as long as we enforce equal heights and widths. using hard coded numbers will make it difficult in the future to make the design responsive.